README.md

```diff
@@ -2,7 +2,11 @@
 
 [![npm version](https://badge.fury.io/js/rendition.svg)](http://badge.fury.io/js/rendition)
 [![Build Status](https://travis-ci.org/resin-io-modules/rendition.svg?branch=master)](https://travis-ci.org/resin-io-modules/rendition)
+[![Dependency Status](https://img.shields.io/david/resin-io-modules/rendition.svg)](https://david-dm.org/resin-io-modules/rendition)
```

Comment:
This currently puts a big badge saying 'insecure' at the top of our readme, which doesn't seem ideal. Looks like our moment version needs a bump.

Reply:
Yep, I noticed that myself, I'll follow up with another PR to bump the version ASAP